PendingReleaseNotes.md

```diff
@@ -41,4 +53,18 @@
   An upgraded operator will create the necessary service account, role, and role bindings to enable this.
 - API
   - The API pod now uses RBAC permissions that are scoped only to the namespace it is running in.
-  An upgraded operator will create the necessary service account, role, and role bindings to enable this.
\ No newline at end of file
+  An upgraded operator will create the necessary service account, role, and role bindings to enable this.
+- Filesystem
+  - The Rook Flexvolume uses the `mds_namespace` option to specify a cephFS. This is only available on Kernel v4.7 or newer. On older kernel, revert back to using the CephFS Kubernetes plugin to consume the filesystem as describe in this [doc](https://github.com/rook/rook/blob/release-0.5/Documentation/k8s-filesystem.md#deploy-the-application)
+
+## Known Issues
+
+- Rook Flexvolume
+  - For Kubernetes cluster 1.7.x and older, Kubelet will need to be restarted in order to load the new flexvolume. This has been resolve in K8S 1.8. For more information about the requirements, refer to this [doc](/Documentation/kubernetes.md#restart-kubelet)
+  - For Kubernetes cluster 1.6.x, the attacher/detacher controller needs to be disabled in order to load the new Flexvolume. This is caused by a [regression](https://github.com/kubernetes/features/blob/master/release-1.6/release-notes-draft.md#volume) on 1.6.x.  For more information about the requirements, refer to this [doc](Documentation/kubernetes.md#disable-attacher-detacher-controller)
+  - For CoreOS and Rancher Kubernetes, the Flexvolume plugin dir will need to be specified to be different than the default. Refer to [Flexvolume configuration pre-reqs](/Documentation/k8s-pre-reqs.md#coreos-container-linux)
+
+## Deprecations
+
+- Rook Standalone
+  - Standalone mode has been disabled and is no longer supported. A Kubernetes environment is required to run Rook.
```

Comment:
We should deprecate the rook API and rookctl too.

Reply:
Shouldn't standalone be in the breaking changes section? Something that's deprecated is still expected to work, but standalone is dead.

Reply:
It is.. i just added it there

Reply:
i would make a distinction between how rook-api and rookctl are deprecated (Planned to be removed but not removed yet) and how Standalone is completely removed now.

Reply:
@jbw976 that is why the standalone deprecation is also in `Breaking Changes`.

Reply:
so why is standalone mentioned in this section? it's already in the breaking changes section.

Reply:
Standalone is in this section because it has been deprecated. It is also mentioned in the `breaking changes` section because it has been yanked out.

Reply:
deprecated means it still works. There is nothing about standalone that still works

Reply:
how about we put the main content/explanation into breaking changes, because that's what it is.  it's gone and if you were using it then that breaks you.  Then in the deprecations section, we can just include a smaller note such as "As mentioned above, support for Standalone mode has been removed completely".  thoughts?